src/test/scala/no/ndla/imageapi/service/WriteServiceTest.scala

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Mulig det bare er dok'en over som er feil. Men hvis det faktisk er støtte for å laste opp flere filer samtidig, savner jeg tester for det.